_data/projects/krayin.yml

```
@@ -0,0 +1,9 @@
+name: krayincrm
+desc: Krayin CRM is a hand tailored CRM framework built on some of the hottest opensource technologies such as Laravel (a PHP framework) and Vue.js a progressive Javascript framework.
+site: https://krayincrm.com/
+tags:
+  - opensource-crm
+  - laravel-crm
+upforgrabs:
+  name: help wanted
```

:art: this is case-sensitive

```suggestion
  name: Help Wanted
```